### ISSUE TITLE
Updater bug

### ISSUE THREAD

**Issue by TheDGOfficial** [LifeMC/LifeSkript#1]
**Describe the bug**
The latest version of the Skript Fixes is V10 and updater shows a "... V10 is out" style warning even in V10.

**To Reproduce**
Just open server with V10 version. Updater shows a "... V10 is out" style warning.

**Expected behavior**
The V10 is already the latest version for now. Updater should show a "... you are running latest version ..." style message.

**Screenshots**
Skipped (N/A)

**Version information:**
 - Minecraft Version: Any
 - Skript Version: 2.2-Fixes-V10

**Additional information**
The updater outputs after server initialization is below.

```batch
[05:13:41] [Craft Scheduler Thread - 0/INFO]: [Skript] A new version of Skript has been found. Skript ﻿2.2-Fixes-V10 has been released. It's highly recommended to upgrade to the latest skript version.
[05:13:41] [Craft Scheduler Thread - 0/INFO]: [Skript] You can download the latest Skript version here: https://github.com/LifeMC/LifeSkript/releases/latest/
```

**Comment by TheDGOfficial** [comment]
We are already trimmed the string here

https://github.com/LifeMC/LifeSkript/blob/b6518d024abb868231be02a0847f3551da2dde46/src/main/java/ch/njol/skript/Skript.java#L456

So i think this not a whitespace issue. The current updater is written by Mirreski. I try to replace it with my own http connection system.

**Comment by TheDGOfficial** [comment]
It seems to be a locale specific issue with "equalsIgnoreCase" method in my computer.

**Comment by TheDGOfficial** [comment]
Oh... I found a whitespace in the latest version. Now actually fixing the issue.

**Comment by TheDGOfficial** [comment]
I forgot to include "fixes" on commit message, my mistake, i'm manually closing this now.
The fix commit available in b6518d0
